resources/web/wwi/protoVisualizer/Vrml.js

```diff
@@ -436,10 +436,10 @@ export class SFNode extends SingleValue {
       return false;
 
     for (const [parameterName, parameter] of this.value.parameters) {
-      if (!other.parameters.has(parameterName))
+      if (!other.value.parameters.has(parameterName))
         return false;
 
-      const otherParameter = other.parameters.get(parameterName);
+      const otherParameter = other.value.parameters.get(parameterName);
```

Comment:
I'm not sure to understand why this change was necessary. `other` is supposed to be a Node instance (which can be a PROTO or a base node), and in the case it's a PROTO, then the parameters have to be retrieved directly from it. Accessing the value of the node entails means we are retrieving the parameters of the derived PROTO. It might work, but intuitively seems the wrong approach. Do you remember why it was necessary to do so?

Reply:
Because `other.parameters` was undefined in some cases

Reply:
that should be the case only if a PROTO without parameters is instantiated, was it with the officechair or another proto?

Reply:
Yes, error when trying to download it

Reply:
since it's a SFNode, then yes, the `.value` is necessary, I thought we were passing the value itself already